pkg/proxy/accesslog/log.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/cilium/cilium/pkg/logging"
 	"github.com/cilium/cilium/pkg/logging/logfields"
 
+	"github.com/sirupsen/logrus"
```

Comment:
Same comment as for `builder.go`, this is imposing a new dependency on third-parties using the `pkg/proxy/accesslog` package. Could this be avoided?